refactor: increase chance of bandit thug to have no helmet

Uberbagel opened the pull request.

Uberbagel commented:
bump

LordMidas reviewed (approved):
The commit message has a typo: `change` instead of `chance`. Please fix that before merging.

Uberbagel commented:
> The commit message has a typo: `change` instead of `chance`. Please fix that before merging. I've changed the title of the PR, but not the commit.

fixed
